rpa-bot/1.1/modules/ROOT/pages/configuration.adoc

```diff
@@ -81,33 +79,29 @@ Enables authentication via SmartCard within the Secure Session.
+If the group policy _Always prompt for password upon connection_ is enabled, the password must be entered twice when logging into the RDP session. This might prevent the Secure Session from running smoothly, especially under Windows 2010 and Windows Server 2016. If this is the case, activate this option so that the credentials can be entered correctly in the second query.
 +
-In this context, ensure that you have selected the option *Allow connections only from computers running Remote Desktop with Network Level Authentication (recommended)* in the Windows System Properties.
+In this context, ensure that you have selected the option *Allow connections only from computers running Remote Desktop with Network Level Authentication (recommended)* in the Windows system properties.
 
 === Test Settings
 
-You have the possibility to test your connection settings for the Secure Session with the *TEST SETTINGS* button. Clicking this button opens the pop-up window Test Settings, in which you can enter test parameters for the Secure Session.
+Optionally, click *TEST SETTINGS* to test your connection settings for the Secure Session. In the *TEST SETTINGS* dialog, enter test parameters for the Secure Session.
 
 image:bot-configurator-test-settings.png[The RPA Bot configurator application showing the Test Settings window, 50%, 50%]
 
-If you click on *TEST*, the application starts a non-operational Secure Session in which you can see your parameters for the Secure Session listed. Click the *Logout* button to end this non-operational Secure Session.
+If you click *TEST*, the application starts a non-operational Secure Session in which you can see your parameters for the Secure Session listed. Click *Logout* to end this non-operational Secure Session.
 
 == Connection Settings
 
 image:bot-configurator-connection-settings.png[The RPA Bot configurator application showing the Connection settings panel, 50%, 50%]
 
 * *Connection settings*
 +
-Here you can define the method how RPA Bot components (Robot, StSec and StSem) communicate among themselves. These settings are only relevant for Secure Sessions. Therefore, you can choose between the Named Pipes protocol or TCP. Named Pipes is selected by default, as an uncomplicated method. In some cases, when using TCP, the firewall can block the connection even though it is an exclusive internal computer process.
+Here you can define how RPA Bot components (Robot, StSec and StSem) communicate among themselves. These settings are only relevant for Secure Sessions. You can choose the Named Pipes protocol or TCP. Named Pipes is selected by default, as an uncomplicated method. In some cases, when using TCP, the firewall can block the connection even though it is an exclusive internal computer process.
```

Comment:
do these 3 need backticks? (Robot, StSec and StSem)

Reply:
only relevant for -> relevant only for?

Reply:
re "as an uncomplicated method" do we mean that it is a less complicated method than TCP?